package.json

```diff
@@ -52,5 +54,13 @@
   },
   "engines": {
     "node": "^14.21 || ^16.20 || ^18.16 || >=20"
-  }
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "@lavamoat/preinstall-always-fail": false,
+      "ethereumjs-util>keccak": false,
+      "ethereumjs-util>secp256k1": false
+    }
+  },
+  "packageManager": "yarn@3.6.3"
```

Comment:
Using `yarn@3.3.0` here to align with core would make migration easier, unless there are updates in the latest version that we want to introduce?

Reply:
It should be trivial for `core` to update to a more recent Yarn version too.